.github/mergify.yml

```diff
@@ -8,19 +8,8 @@ defaults:
         {{ body | get_section("## Description", "") }}
 
         Pull-Request: #{{ number }}.
-        {# Here comes some fancy Jinja2 stuff for correctly attributing co-authorship: #}
-        {%- set _ = 0 -%}
-        {%- for commit in commits -%}
-          {%- if commit.parents|length != 1 -%}
-            {%- set _ = commit.update({'merge': true}) -%}
-          {%- else -%}
-            {%- set _ = commit.update({'merge': false}) -%}
-          {%- endif -%}
-        {%- endfor -%}
-        {%- for commit in (commits | rejectattr("merge") | unique(False, 'email_author')) | rejectattr("author", "==", author) -%}
-        Co-authored-by: {{ commit.author }} <{{ commit.email_author }}>
-        {% endfor %}
-        {# GitHub requires that the `Co-authored-by` lines are AT THE VERY END of a commit, hence nothing must come after this. #}
+
+        {{ body | get_section("## Attributions", "") }}
```

Comment:
Want to make this part of our pull request template?

https://github.com/libp2p/rust-libp2p/blob/master/.github/pull_request_template.md

Reply:
It is only temporary so I am not too fussed.